### ISSUE TITLE
Additions to the API involving how many users have opted into receiving vote timer notifications. 

### ISSUE THREAD

**Issue by Nathan-Webb** [DiscordBotList/issues#200]
#### Suggestion

There should be an API endpoint to view how many people have opted-in to receive notifications when they can vote. It would be useful to Bot Devs because they can use this number to measure how many people actively vote, and use this number for analytics on how their bot is doing in terms of community engagement. 

There should also be a way to check if a user has signed up to be notified when they can vote. This would give bot devs the ability to reward those who opted in.

**Comment by devakira** [comment]
There could also be a webhook that triggers when the user is able to vote again, then a message could be sent through Discord reminding the user that they can vote.

**Comment by Nathan-Webb** [comment]
> There could also be a webhook that triggers when the user is able to vote again, then a message could be sent through Discord reminding the user that they can vote.

maybe make a new issue? @devakira

**Comment by devakira** [comment]
@nathan-webb my suggestion still falls under the general "extensions to the vote API", so it might get marked as a duplicate of I make a new issue.

**Comment by Chewsterchew** [comment]
@devakira Not really. This issue, from what I read is asking for the amount of people who opted in to be shown in the API. However, yours is suggesting a webhook, which isn't about the API, it's something different now. I can guarantee a issue dubbed "Add webhooks for vote reminder" would not be closed.
In my bot, I just took the webhook and scheduled a vote reminder in 12 hours if they opted in to it (through the bot), but it would be cool to webhook vote remind.
